neurom/apps/tests/test_cli.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+from click.testing import CliRunner
+from mock import MagicMock, patch
+from nose.tools import assert_equal
+
+from neurom.apps.cli import cli
+
+DATA = Path(__file__).parent.parent.parent.parent.resolve() / 'test_data'
+
+
+@patch('neurom.apps.cli.plt.show', MagicMock())
```

Comment:
Can we verify that the mocked method was called once? The same applies to another test.

Reply:
Good point